lib/tasks/stimulus_tasks.rake

```diff
@@ -7,6 +7,11 @@ module Stimulus
       system RbConfig.ruby, "./bin/rails", "app:template", "LOCATION=#{File.expand_path("../install/#{path}.rb",  __dir__)}"
     end
 
+    def controllers_path
+      js_root = Rails.application.config.generators.options.dig(:stimulus, :js_root) || "app/javascript"
```

Comment:
Wouldn't js_root always be set, given that you have a default for the class_option? If so, I don't think we need to repeat the default here. And in fact, I'd just inline the entire call in the Rails.root.join.

Reply:
I pushed up this change. LMK what you think. 

However this pans out thanks for the code review 👍